docs/topics/pagination.txt

```diff
@@ -305,3 +305,44 @@ Attributes
 .. attribute:: Page.paginator
 
     The associated :class:`Paginator` object.
+
+
+Custom pagination
```

Comment:
What if there were other "custom" paginators? This heading should be:
```python
 ``EllipsisPaginator`` objects
 =============================
```
Also move it up underneath the section for the main paginator object.